README.md

```diff
@@ -0,0 +1,36 @@
+[![Build and Test](https://github.com/bigladder/lattice/actions/workflows/build-and-test.yaml/badge.svg)](https://github.com/bigladder/lattice/actions/workflows/build-and-test.yaml)
+
+[![Web Documentation](https://github.com/bigladder/lattice/actions/workflows/build-web.yaml/badge.svg)](https://github.com/bigladder/lattice/actions/workflows/build-web.yaml)
+
+Lattice
+===========
+
+A toolkit that provides support for a schema-based building data model framework, currently under development as ASHRAE Standard 232P. Lattice performs data model validation, intermediate schema generation, and data file validation for YAML data models compliant with ASHRAE 232P and ASHRAE 205.
+
+
+Installing the Toolkit
+--------------------
+
+We are currently supporting python 3.7 - 3.10.
+
+To install and use the Lattice project, you must have a supported version of Python. We also use the [poetry](https://python-poetry.org/) tool to manage Lattice's dependencies. Follow the [instructions](https://python-poetry.org/docs/#installation) to install poetry on your specific platform.
```

Comment:
Recommend installing via pip instead? Poetry is really only needed for developers, which should be covered in a separate section.